app/src/main/res/drawable/ic_facility_reassignment.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This icon frame doesn't match the one in the design. We need to download the icon and frame, instead of just the icon glyph.